rs/consensus/src/consensus/validator.rs

```diff
@@ -882,7 +882,7 @@ impl Validator {
         let max_height = notarization_height.increment();
         let range = HeightRange::new(finalized_height.increment(), max_height);
         // Collect the min of validated block proposal ranks in the range.
```

Comment:
Shall we mention that we are not considering disqualified blocks here?

Reply:
done